internal/hcsoci/hcsdoc_lcow.go

```diff
@@ -65,8 +65,19 @@ func createLCOWSpec(coi *createOptionsInternal) (*specs.Spec, error) {
 	return spec, nil
 }
 
+// This is identical to hcsschema.ComputeSystem but HostedSystem is an LCOW specific type - the schema docs only include WCOW.
+type linuxComputeSystem struct {
+	Owner                             string                    `json:"Owner,omitempty"`
+	SchemaVersion                     *hcsschema.Version        `json:"SchemaVersion,omitempty"`
+	HostingSystemId                   string                    `json:"HostingSystemId,omitempty"`
+	HostedSystem                      *linuxHostedSystem        `json:"HostedSystem,omitempty"`
+	Container                         *hcsschema.Container      `json:"Container,omitempty"`
+	VirtualMachine                    *hcsschema.VirtualMachine `json:"VirtualMachine,omitempty"`
+	ShouldTerminateOnLastHandleClosed bool                      `json:"ShouldTerminateOnLastHandleClosed,omitempty"`
+}
```

Comment:
I should consider moving this to internal/lcow/types.go.

Reply:
Any reason not to create a `genericComputeSystem` that defines the shared poriton and then a `windows`/`linux` that adds the HostedSystem in each type?